lib/common.sh

```diff
@@ -46,7 +46,14 @@ fi
 
 export EXTERNAL_SUBNET="192.168.111.0/24"
 
-export SSH_PUB_KEY=~/.ssh/id_rsa.pub
+export SSH_KEY_PATH=~/.ssh/id_rsa
+export SSH_PUB_KEY_PATH="${SSH_KEY_PATH}.pub"
```

Comment:
These are hard-coded and won't allow override from the config as in `config_example.sh`

Should be like this I think?

```
export SSH_KEY_PATH=${SSH_KEY_PATH:-~/.ssh/id_rsa}
export SSH_PUB_KEY_PATH=${SSH_PUB_KEY_PATH:-${SSH_KEY_PATH}.pub}
```

Reply:
indeed, I will fix this, thank you